cogs5e/models/automation/effects/damage.py

```diff
@@ -109,9 +110,14 @@ def run(self, autoctx):
         # evaluate damage
         dmgroll = d20.roll(dice_ast)
 
-        # magic arg (#853), magical effect (#1063)
```

Comment:
nit: please keep the related issue numbers in the comment! I've opened AFR-825 as #1544